modules/selection/test/spec.coffee

```diff
@@ -1078,15 +1078,15 @@ describe 'Selection Api', ->
     div = hx.detached('div')
     called = false
     div.on 'click', (e) -> called = true
-    div.node().__hx__.eventEmitter.emit('click', {})
+    testHelpers.fakeNodeEvent(div.node(), 'click')({})
```

Comment:
One of the tests was failing for selection and whilst I was investigating I thought I might as well update to use `testHelpers`